src/structures/MessageEmbed.js

```diff
@@ -314,8 +315,17 @@ class MessageEmbed {
    * @param {string} value The value of this field
    * @param {boolean} [inline=false] If this field will be displayed inline
    * @returns {MessageEmbed}
+   * @deprecated This method is a wrapper for {@link MessageEmbed.addFields}, use that instead.
```

Comment:
The `.` references static methods.

```suggestion
   * @deprecated This method is a wrapper for {@link MessageEmbed#addFields}. Use that instead.
```